[2024-12-19T07:50:52.003+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T07:50:52.011+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v04.Push_Name scheduled__2024-12-18T00:00:00+00:00 [queued]>
[2024-12-19T07:50:52.016+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v04.Push_Name scheduled__2024-12-18T00:00:00+00:00 [queued]>
[2024-12-19T07:50:52.016+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T07:50:52.023+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Push_Name> on 2024-12-18 00:00:00+00:00
[2024-12-19T07:50:52.029+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1040) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T07:50:52.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v04', 'Push_Name', 'scheduled__2024-12-18T00:00:00+00:00', '--job-id', '2197', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_python_operator.py', '--cfg-path', '/tmp/tmpnuhqngym']
[2024-12-19T07:50:52.030+0000] {standard_task_runner.py:72} INFO - Started process 1049 to run task
[2024-12-19T07:50:52.030+0000] {standard_task_runner.py:105} INFO - Job 2197: Subtask Push_Name
[2024-12-19T07:50:52.055+0000] {task_command.py:467} INFO - Running <TaskInstance: our_dag_with_python_operator_v04.Push_Name scheduled__2024-12-18T00:00:00+00:00 [running]> on host ad6490b5d50c
[2024-12-19T07:50:52.093+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v04' AIRFLOW_CTX_TASK_ID='Push_Name' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-18T00:00:00+00:00'
[2024-12-19T07:50:52.093+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T07:50:52.109+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-19T07:50:52.112+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T07:50:52.113+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v04, task_id=Push_Name, run_id=scheduled__2024-12-18T00:00:00+00:00, execution_date=20241218T000000, start_date=20241219T075052, end_date=20241219T075052
[2024-12-19T07:50:52.130+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T07:50:52.139+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T07:50:52.139+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
